app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (VaccineError, NotWearingMaskError)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    vaccinated_friends = 0
+    friends_without_mask = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+            vaccinated_friends += 1
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            friends_without_mask += 1
+
+    if friends_without_mask > 0:
+        return f"Friends should buy {friends_without_mask} masks"
+    else:
```

Comment:
There is no need for `else` since the code will not execute after the `return` statement anyway. Please, remove the redundant `else`